.go-version

```diff
@@ -0,0 +1 @@
+1.22.3
```

Comment:
As a library, let's try not to use the latest go minor version, otherwise the applications, which depend on gofail, will have to bump their go version to the latest go minor version as well
```suggestion
1.21.10
```